src/test/java/org/sagebionetworks/web/unitclient/widget/entity/file/AddToDownloadListTest.java

```diff
@@ -111,6 +119,97 @@ public void setUp() throws Exception {
 		when(mockAuthController.isLoggedIn()).thenReturn(true);
 	}
 
+	@Test
```

Comment:
Copy the old tests, but put it in alpha mode to verify the Download List V2 request.